x/amm/keeper/fee.go

```diff
@@ -16,9 +19,44 @@ func PortionCoins(coins sdk.Coins, portion sdk.Dec) sdk.Coins {
 	return portionCoins
 }
 
-func (k Keeper) OnCollectFee(ctx sdk.Context, pool types.Pool, fee sdk.Coins) {
-	// TODO: transfer 60% to LPs through commitment module
-	// TODO: transfer 30% to stakers through incentive module
-	// keep remaining 10% on fees treasury
-	// TODO: swap fees to pool FeeDenom (this is normally USDC)
+func (k Keeper) OnCollectFee(ctx sdk.Context, pool types.Pool, fee sdk.Coins) error {
+	poolRevenueAddress := types.NewPoolRevenueAddress(pool.PoolId)
```

Comment:
@jelysn I guess we have agreed on defining this function in incentive module side, didn't we?